File: Estrutura_Repetição/C++/Ex1117/Main.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    float media = 0;
    int nota_valida = 0;

    while (true) {
        float nota;
        cin >> nota;

        if (nota >= 0 && nota <= 10) {
            nota_valida++;
            media += nota;
            if (nota_valida == 2) {
                cout << fixed << setprecision(2) << "media = " << media / 2 << endl;
                break;
            }
        } else {
            cout << "nota invalida" << endl;
        }
    }

    return 0;
}

File: Estrutura_Repetição/C#/Ex1117/Main.cs
using System;

class MainClass {
    static void Main(string[] args) {
        float media = 0;
        int notaValida = 0;

        while (true) {
            float nota = float.Parse(Console.ReadLine());

            if (nota >= 0 && nota <= 10) {
                notaValida++;
                media += nota;
                if (notaValida == 2) {
                    Console.WriteLine("media = " + (media / 2).ToString("0.00"));
                    break;
                }
            } else {
                Console.WriteLine("nota invalida");
            }
        }
    }
}